### PR TITLE
fix(components): [table-v2] header checkbox cover fixedHeader when scroll

### LINKED ISSUE #18163: [Component] [table-v2] El-VirtualTable select checkbox of column header will cover the fixed column when scroll horizontally.
Bug Type: **`Component`**

## Environment

- Vue Version: `3.4.38`
- Element Plus Version: `2.8.0`
- Browser / OS: `Chrome (128.0.6613.85)`
- Build Tool: `Vite`

### Related Component

- `el-table-v2`

### Reproduction Link

[Element Plus Playground](https://element-plus.run/#eyJBcHAudnVlIjoiPHRlbXBsYXRlPlxuICA8ZGl2IHN0eWxlPVwiaGVpZ2h0OiA0MDBweFwiPlxuICAgIDxlbC1hdXRvLXJlc2l6ZXI+XG4gICAgICA8dGVtcGxhdGUgI2RlZmF1bHQ9XCJ7IGhlaWdodCwgd2lkdGggfVwiPlxuICAgICAgICA8ZWwtdGFibGUtdjJcbiAgICAgICAgICA6Y29sdW1ucz1cImNvbHVtbnNcIlxuICAgICAgICAgIDpkYXRhPVwiZGF0YVwiXG4gICAgICAgICAgOndpZHRoPVwid2lkdGhcIlxuICAgICAgICAgIDpoZWlnaHQ9XCJoZWlnaHRcIlxuICAgICAgICAgIGZpeGVkXG4gICAgICAgIC8+XG4gICAgICA8L3RlbXBsYXRlPlxuICAgIDwvZWwtYXV0by1yZXNpemVyPlxuICA8L2Rpdj5cbjwvdGVtcGxhdGU+XG5cbjxzY3JpcHQgbGFuZz1cInRzeFwiIHNldHVwPlxuaW1wb3J0IHsgcmVmLCB1bnJlZiB9IGZyb20gJ3Z1ZSdcbmltcG9ydCB7IEVsQ2hlY2tib3ggfSBmcm9tICdlbGVtZW50LXBsdXMnXG5cbmltcG9ydCB0eXBlIHsgRnVuY3Rpb25hbENvbXBvbmVudCB9IGZyb20gJ3Z1ZSdcbmltcG9ydCB0eXBlIHsgQ2hlY2tib3hWYWx1ZVR5cGUsIENvbHVtbiB9IGZyb20gJ2VsZW1lbnQtcGx1cydcblxudHlwZSBTZWxlY3Rpb25DZWxsUHJvcHMgPSB7XG4gIHZhbHVlOiBib29sZWFuXG4gIGludGVybWVkaWF0ZT86IGJvb2xlYW5cbiAgb25DaGFuZ2U6ICh2YWx1ZTogQ2hlY2tib3hWYWx1ZVR5cGUpID0+IHZvaWRcbn1cblxuY29uc3QgU2VsZWN0aW9uQ2VsbDogRnVuY3Rpb25hbENvbXBvbmVudDxTZWxlY3Rpb25DZWxsUHJvcHM+ID0gKHtcbiAgdmFsdWUsXG4gIGludGVybWVkaWF0ZSA9IGZhbHNlLFxuICBvbkNoYW5nZSxcbn0pID0+IHtcbiAgcmV0dXJuIChcbiAgICA8RWxDaGVja2JveFxuICAgICAgb25DaGFuZ2U9e29uQ2hhbmdlfVxuICAgICAgbW9kZWxWYWx1ZT17dmFsdWV9XG4gICAgICBpbmRldGVybWluYXRlPXtpbnRlcm1lZGlhdGV9XG4gICAgLz5cbiAgKVxufVxuXG5jb25zdCBnZW5lcmF0ZUNvbHVtbnMgPSAobGVuZ3RoID0gMTAsIHByZWZpeCA9ICdjb2x1bW4tJywgcHJvcHM/OiBhbnkpID0+XG4gIEFycmF5LmZyb20oeyBsZW5ndGggfSkubWFwKChfLCBjb2x1bW5JbmRleCkgPT4gKHtcbiAgICAuLi5wcm9wcyxcbiAgICBrZXk6IGAke3ByZWZpeH0ke2NvbHVtbkluZGV4fWAsXG4gICAgZGF0YUtleTogYCR7cHJlZml4fSR7Y29sdW1uSW5kZXh9YCxcbiAgICB0aXRsZTogYENvbHVtbiAke2NvbHVtbkluZGV4fWAsXG4gICAgZml4ZWQ6IGNvbHVtbkluZGV4ID09IDAgPyB0cnVlIDogZmFsc2UsXG4gICAgd2lkdGg6IDE1MCxcbiAgfSkpXG5cbmNvbnN0IGdlbmVyYXRlRGF0YSA9IChcbiAgY29sdW1uczogUmV0dXJuVHlwZTx0eXBlb2YgZ2VuZXJhdGVDb2x1bW5zPixcbiAgbGVuZ3RoID0gMjAwLFxuICBwcmVmaXggPSAncm93LSdcbikgPT5cbiAgQXJyYXkuZnJvbSh7IGxlbmd0aCB9KS5tYXAoKF8sIHJvd0luZGV4KSA9PiB7XG4gICAgcmV0dXJuIGNvbHVtbnMucmVkdWNlKFxuICAgICAgKHJvd0RhdGEsIGNvbHVtbiwgY29sdW1uSW5kZXgpID0+IHtcbiAgICAgICAgcm93RGF0YVtjb2x1bW4uZGF0YUtleV0gPSBgUm93ICR7cm93SW5kZXh9IC0gQ29sICR7Y29sdW1uSW5kZXh9YFxuICAgICAgICByZXR1cm4gcm93RGF0YVxuICAgICAgfSxcbiAgICAgIHtcbiAgICAgICAgaWQ6IGAke3ByZWZpeH0ke3Jvd0luZGV4fWAsXG4gICAgICAgIGNoZWNrZWQ6IGZhbHNlLFxuICAgICAgICBwYXJlbnRJZDogbnVsbCxcbiAgICAgIH1cbiAgICApXG4gIH0pXG5cbmNvbnN0IGNvbHVtbnM6IENvbHVtbjxhbnk+W10gPSBnZW5lcmF0ZUNvbHVtbnMoMTApXG5jb2x1bW5zLnVuc2hpZnQoe1xuICBrZXk6ICdzZWxlY3Rpb24nLFxuICB3aWR0aDogNTAsXG4gIFxuICBjZWxsUmVuZGVyZXI6ICh7IHJvd0RhdGEgfSkgPT4ge1xuICAgIGNvbnN0IG9uQ2hhbmdlID0gKHZhbHVlOiBDaGVja2JveFZhbHVlVHlwZSkgPT4gKHJvd0RhdGEuY2hlY2tlZCA9IHZhbHVlKVxuICAgIHJldHVybiA8U2VsZWN0aW9uQ2VsbCB2YWx1ZT17cm93RGF0YS5jaGVja2VkfSBvbkNoYW5nZT17b25DaGFuZ2V9IC8+XG4gIH0sXG5cbiAgaGVhZGVyQ2VsbFJlbmRlcmVyOiAoKSA9PiB7XG4gICAgY29uc3QgX2RhdGEgPSB1bnJlZihkYXRhKVxuICAgIGNvbnN0IG9uQ2hhbmdlID0gKHZhbHVlOiBDaGVja2JveFZhbHVlVHlwZSkgPT5cbiAgICAgIChkYXRhLnZhbHVlID0gX2RhdGEubWFwKChyb3cpID0+IHtcbiAgICAgICAgcm93LmNoZWNrZWQgPSB2YWx1ZVxuICAgICAgICByZXR1cm4gcm93XG4gICAgICB9KSlcbiAgICBjb25zdCBhbGxTZWxlY3RlZCA9IF9kYXRhLmV2ZXJ5KChyb3cpID0+IHJvdy5jaGVja2VkKVxuICAgIGNvbnN0IGNvbnRhaW5zQ2hlY2tlZCA9IF9kYXRhLnNvbWUoKHJvdykgPT4gcm93LmNoZWNrZWQpXG5cbiAgICByZXR1cm4gKFxuICAgICAgPFNlbGVjdGlvbkNlbGxcbiAgICAgICAgdmFsdWU9e2FsbFNlbGVjdGVkfVxuICAgICAgICBpbnRlcm1lZGlhdGU9e2NvbnRhaW5zQ2hlY2tlZCAmJiAhYWxsU2VsZWN0ZWR9XG4gICAgICAgIG9uQ2hhbmdlPXtvbkNoYW5nZX1cbiAgICAgIC8+XG4gICAgKVxuICB9LFxufSlcblxuY29uc3QgZGF0YSA9IHJlZihnZW5lcmF0ZURhdGEoY29sdW1ucywgMjAwKSlcbjwvc2NyaXB0PlxuIiwiZWxlbWVudC1wbHVzLmpzIjoiaW1wb3J0IHsgZ2V0Q3VycmVudEluc3RhbmNlIH0gZnJvbSAndnVlJ1xuaW1wb3J0IEVsZW1lbnRQbHVzIGZyb20gJ2VsZW1lbnQtcGx1cydcblxubGV0IGluc3RhbGxlZCA9IGZhbHNlXG5hd2FpdCBsb2FkU3R5bGUoKVxuXG5leHBvcnQgZnVuY3Rpb24gc2V0dXBFbGVtZW50UGx1cygpIHtcbiAgaWYgKGluc3RhbGxlZCkgcmV0dXJuXG4gIGNvbnN0IGluc3RhbmNlID0gZ2V0Q3VycmVudEluc3RhbmNlKClcbiAgaW5zdGFuY2UuYXBwQ29udGV4dC5hcHAudXNlKEVsZW1lbnRQbHVzKVxuICBpbnN0YWxsZWQgPSB0cnVlXG59XG5cbmV4cG9ydCBmdW5jdGlvbiBsb2FkU3R5bGUoKSB7XG4gIGNvbnN0IHN0eWxlcyA9IFsnaHR0cHM6Ly9mYXN0bHkuanNkZWxpdnIubmV0L25wbS9lbGVtZW50LXBsdXNAbGF0ZXN0L2Rpc3QvaW5kZXguY3NzJywgJ2h0dHBzOi8vZmFzdGx5LmpzZGVsaXZyLm5ldC9ucG0vZWxlbWVudC1wbHVzQGxhdGVzdC90aGVtZS1jaGFsay9kYXJrL2Nzcy12YXJzLmNzcyddLm1hcCgoc3R5bGUpID0+IHtcbiAgICByZXR1cm4gbmV3IFByb21pc2UoKHJlc29sdmUsIHJlamVjdCkgPT4ge1xuICAgICAgY29uc3QgbGluayA9IGRvY3VtZW50LmNyZWF0ZUVsZW1lbnQoJ2xpbmsnKVxuICAgICAgbGluay5yZWwgPSAnc3R5bGVzaGVldCdcbiAgICAgIGxpbmsuaHJlZiA9IHN0eWxlXG4gICAgICBsaW5rLmFkZEV2ZW50TGlzdGVuZXIoJ2xvYWQnLCByZXNvbHZlKVxuICAgICAgbGluay5hZGRFdmVudExpc3RlbmVyKCdlcnJvcicsIHJlamVjdClcbiAgICAgIGRvY3VtZW50LmJvZHkuYXBwZW5kKGxpbmspXG4gICAgfSlcbiAgfSlcbiAgcmV0dXJuIFByb21pc2UuYWxsU2V0dGxlZChzdHlsZXMpXG59XG4iLCJ0c2NvbmZpZy5qc29uIjoie1xuICBcImNvbXBpbGVyT3B0aW9uc1wiOiB7XG4gICAgXCJ0YXJnZXRcIjogXCJFU05leHRcIixcbiAgICBcImpzeFwiOiBcInByZXNlcnZlXCIsXG4gICAgXCJtb2R1bGVcIjogXCJFU05leHRcIixcbiAgICBcIm1vZHVsZVJlc29sdXRpb25cIjogXCJCdW5kbGVyXCIsXG4gICAgXCJ0eXBlc1wiOiBbXCJlbGVtZW50LXBsdXMvZ2xvYmFsLmQudHNcIl0sXG4gICAgXCJhbGxvd0ltcG9ydGluZ1RzRXh0ZW5zaW9uc1wiOiB0cnVlLFxuICAgIFwiYWxsb3dKc1wiOiB0cnVlLFxuICAgIFwiY2hlY2tKc1wiOiB0cnVlXG4gIH0sXG4gIFwidnVlQ29tcGlsZXJPcHRpb25zXCI6IHtcbiAgICBcInRhcmdldFwiOiAzLjNcbiAgfVxufVxuIiwiUGxheWdyb3VuZE1haW4udnVlIjoiPHNjcmlwdCBzZXR1cD5cbmltcG9ydCBBcHAgZnJvbSAnLi9BcHAudnVlJ1xuaW1wb3J0IHsgc2V0dXBFbGVtZW50UGx1cyB9IGZyb20gJy4vZWxlbWVudC1wbHVzLmpzJ1xuc2V0dXBFbGVtZW50UGx1cygpXG48L3NjcmlwdD5cblxuPHRlbXBsYXRlPlxuICA8QXBwIC8+XG48L3RlbXBsYXRlPlxuIiwiaW1wb3J0LW1hcC5qc29uIjoie1xuICBcImltcG9ydHNcIjoge1xuICAgIFwidnVlXCI6IFwiaHR0cHM6Ly9mYXN0bHkuanNkZWxpdnIubmV0L25wbS9AdnVlL3J1bnRpbWUtZG9tQGxhdGVzdC9kaXN0L3J1bnRpbWUtZG9tLmVzbS1icm93c2VyLmpzXCIsXG4gICAgXCJAdnVlL3NoYXJlZFwiOiBcImh0dHBzOi8vZmFzdGx5LmpzZGVsaXZyLm5ldC9ucG0vQHZ1ZS9zaGFyZWRAbGF0ZXN0L2Rpc3Qvc2hhcmVkLmVzbS1idW5kbGVyLmpzXCIsXG4gICAgXCJlbGVtZW50LXBsdXNcIjogXCJodHRwczovL2Zhc3RseS5qc2RlbGl2ci5uZXQvbnBtL2VsZW1lbnQtcGx1c0BsYXRlc3QvZGlzdC9pbmRleC5mdWxsLm1pbi5tanNcIixcbiAgICBcImVsZW1lbnQtcGx1cy9cIjogXCJodHRwczovL2Zhc3RseS5qc2RlbGl2ci5uZXQvbnBtL2VsZW1lbnQtcGx1c0BsYXRlc3QvXCIsXG4gICAgXCJAZWxlbWVudC1wbHVzL2ljb25zLXZ1ZVwiOiBcImh0dHBzOi8vZmFzdGx5LmpzZGVsaXZyLm5ldC9ucG0vQGVsZW1lbnQtcGx1cy9pY29ucy12dWVAMi9kaXN0L2luZGV4Lm1pbi5qc1wiXG4gIH0sXG4gIFwic2NvcGVzXCI6IHt9XG59IiwiX28iOnt9fQ==)

### Steps to reproduce

Scroll the virtual table horizontal scroll bar, and watch the row selector checkbox of column header. You will find the checkbox of column header will cover fixed column. The checkboxes in rows do not have this problem.

### What is Expected?

checkbox of column header will not cover fixed column

### What is actually happening?

checkbox of column header will cover fixed column

### Additional comments

_(empty)_

### PR DESCRIPTION
**Pull request by kimverchan**
closed #18163

Please make sure these boxes are checked before submitting your PR, thank you!

- [x] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [x] Make sure you are merging your commits to `dev` branch.
- [x] Add some descriptions and refer to relative issues for your PR.

### CODE REVIEW

File: packages/theme-chalk/src/table-v2.scss [hunk 1]
```diff
@@ -79,6 +79,10 @@
     .#{$namespace}-vl__vertical {
       z-index: 2;
     }
+    
+    @include e('header') {
```
[the remark this answers is not in the document]
Comment (kimverchan) [reply packages/theme-chalk/src/table-v2.scss]: I have made changes as you suggested

[the remark this answers is not in the document]
Comment (kimverchan) [reply packages/theme-chalk/src/table-v2.scss]: @warmthsea 
I have made changes as you suggested

**Review by warmthsea — APPROVED** [review]

File: packages/theme-chalk/src/table-v2.scss [hunk 2]
```diff
@@ -110,6 +110,10 @@
   @include e('header') {
     position: relative;
     overflow: hidden;
+
+    .#{$namespace}-checkbox {
+      z-index: 0;
+    }
```
Comment (btea) [inline packages/theme-chalk/src/table-v2.scss:116]: Maybe we could add `z-index: 1;` on line 89? 🤔

Comment (kimverchan) [reply packages/theme-chalk/src/table-v2.scss:116]: @btea Yes, this is compatible with components other than checkbox, I'll add

Comment (kimverchan) [reply packages/theme-chalk/src/table-v2.scss]: [fixed on left and right exaples](https://element-plus.run/#eyJBcHAudnVlIjoiPHRlbXBsYXRlPlxuICA8ZGl2IHN0eWxlPVwiaGVpZ2h0OiA0MDBweFwiPlxuICAgIDxlbC1hdXRvLXJlc2l6ZXI+XG4gICAgICA8dGVtcGxhdGUgI2RlZmF1bHQ9XCJ7IGhlaWdodCwgd2lkdGggfVwiPlxuICAgICAgICA8ZWwtdGFibGUtdjJcbiAgICAgICAgICA6Y29sdW1ucz1cImNvbHVtbnNcIlxuICAgICAgICAgIDpkYXRhPVwiZGF0YVwiXG4gICAgICAgICAgOndpZHRoPVwid2lkdGhcIlxuICAgICAgICAgIDpoZWlnaHQ9XCJoZWlnaHRcIlxuICAgICAgICAgIGZpeGVkXG4gICAgICAgIC8+XG4gICAgICA8L3RlbXBsYXRlPlxuICAgIDwvZWwtYXV0by1yZXNpemVyPlxuICA8L2Rpdj5cbjwvdGVtcGxhdGU+XG5cbjxzY3JpcHQgbGFuZz1cInRzeFwiIHNldHVwPlxuaW1wb3J0IHsgcmVmLCB1bnJlZiB9IGZyb20gJ3Z1ZSdcbmltcG9ydCB7IEVsQ2hlY2tib3gsIFRhYmxlVjJGaXhlZERpciB9IGZyb20gJ2VsZW1lbnQtcGx1cydcblxuaW1wb3J0IHR5cGUgeyBGdW5jdGlvbmFsQ29tcG9uZW50IH0gZnJvbSAndnVlJ1xuaW1wb3J0IHR5cGUgeyBDaGVja2JveFZhbHVlVHlwZSwgQ29sdW1uIH0gZnJvbSAnZWxlbWVudC1wbHVzJ1xuXG50eXBlIFNlbGVjdGlvbkNlbGxQcm9wcyA9IHtcbiAgdmFsdWU6IGJvb2xlYW5cbiAgaW50ZXJtZWRpYXRlPzogYm9vbGVhblxuICBvbkNoYW5nZTogKHZhbHVlOiBDaGVja2JveFZhbHVlVHlwZSkgPT4gdm9pZFxufVxuXG5jb25zdCBTZWxlY3Rpb25DZWxsOiBGdW5jdGlvbmFsQ29tcG9uZW50PFNlbGVjdGlvbkNlbGxQcm9wcz4gPSAoe1xuICB2YWx1ZSxcbiAgaW50ZXJtZWRpYXRlID0gZmFsc2UsXG4gIG9uQ2hhbmdlLFxufSkgPT4ge1xuICByZXR1cm4gKFxuICAgIDxFbENoZWNrYm94XG4gICAgICBvbkNoYW5nZT17b25DaGFuZ2V9XG4gICAgICBtb2RlbFZhbHVlPXt2YWx1ZX1cbiAgICAgIGluZGV0ZXJtaW5hdGU9e2ludGVybWVkaWF0ZX1cbiAgICAvPlxuICApXG59XG5cbmNvbnN0IGdlbmVyYXRlQ29sdW1ucyA9IChsZW5ndGggPSAxMCwgcHJlZml4ID0gJ2NvbHVtbi0nLCBwcm9wcz86IGFueSkgPT5cbiAgQXJyYXkuZnJvbSh7IGxlbmd0aCB9KS5tYXAoKF8sIGNvbHVtbkluZGV4KSA9PiAoe1xuICAgIC4uLnByb3BzLFxuICAgIGtleTogYCR7cHJlZml4fSR7Y29sdW1uSW5kZXh9YCxcbiAgICBkYXRhS2V5OiBgJHtwcmVmaXh9JHtjb2x1bW5JbmRleH1gLFxuICAgIHRpdGxlOiBgQ29sdW1uICR7Y29sdW1uSW5kZXh9YCxcbiAgICBmaXhlZDogY29sdW1uSW5kZXggPT0gMCB8fCBjb2x1bW5JbmRleCA9PSBsZW5ndGggLTEgPyB0cnVlIDogZmFsc2UsXG4gICAgd2lkdGg6IDE1MCxcbiAgfSkpXG5cbmNvbnN0IGdlbmVyYXRlRGF0YSA9IChcbiAgY29sdW1uczogUmV0dXJuVHlwZTx0eXBlb2YgZ2VuZXJhdGVDb2x1bW5zPixcbiAgbGVuZ3RoID0gMjAwLFxuICBwcmVmaXggPSAncm93LSdcbikgPT5cbiAgQXJyYXkuZnJvbSh7IGxlbmd0aCB9KS5tYXAoKF8sIHJvd0luZGV4KSA9PiB7XG4gICAgcmV0dXJuIGNvbHVtbnMucmVkdWNlKFxuICAgICAgKHJvd0RhdGEsIGNvbHVtbiwgY29sdW1uSW5kZXgpID0+IHtcbiAgICAgICAgcm93RGF0YVtjb2x1bW4uZGF0YUtleV0gPSBgUm93ICR7cm93SW5kZXh9IC0gQ29sICR7Y29sdW1uSW5kZXh9YFxuICAgICAgICByZXR1cm4gcm93RGF0YVxuICAgICAgfSxcbiAgICAgIHtcbiAgICAgICAgaWQ6IGAke3ByZWZpeH0ke3Jvd0luZGV4fWAsXG4gICAgICAgIGNoZWNrZWQ6IGZhbHNlLFxuICAgICAgICBwYXJlbnRJZDogbnVsbCxcbiAgICAgIH1cbiAgICApXG4gIH0pXG5cbmNvbnN0IGNvbHVtbnM6IENvbHVtbjxhbnk+W10gPSBnZW5lcmF0ZUNvbHVtbnMoMTApXG5jb2x1bW5zW2NvbHVtbnMubGVuZ3RoIC0gMV0uZml4ZWQgPSBUYWJsZVYyRml4ZWREaXIuUklHSFRcbmNvbHVtbnMudW5zaGlmdCh7XG4gIGtleTogJ3NlbGVjdGlvbicsXG4gIHdpZHRoOiA1MCxcbiAgXG4gIGNlbGxSZW5kZXJlcjogKHsgcm93RGF0YSB9KSA9PiB7XG4gICAgY29uc3Qgb25DaGFuZ2UgPSAodmFsdWU6IENoZWNrYm94VmFsdWVUeXBlKSA9PiAocm93RGF0YS5jaGVja2VkID0gdmFsdWUpXG4gICAgcmV0dXJuIDxTZWxlY3Rpb25DZWxsIHZhbHVlPXtyb3dEYXRhLmNoZWNrZWR9IG9uQ2hhbmdlPXtvbkNoYW5nZX0gLz5cbiAgfSxcblxuICBoZWFkZXJDZWxsUmVuZGVyZXI6ICgpID0+IHtcbiAgICBjb25zdCBfZGF0YSA9IHVucmVmKGRhdGEpXG4gICAgY29uc3Qgb25DaGFuZ2UgPSAodmFsdWU6IENoZWNrYm94VmFsdWVUeXBlKSA9PlxuICAgICAgKGRhdGEudmFsdWUgPSBfZGF0YS5tYXAoKHJvdykgPT4ge1xuICAgICAgICByb3cuY2hlY2tlZCA9IHZhbHVlXG4gICAgICAgIHJldHVybiByb3dcbiAgICAgIH0pKVxuICAgIGNvbnN0IGFsbFNlbGVjdGVkID0gX2RhdGEuZXZlcnkoKHJvdykgPT4gcm93LmNoZWNrZWQpXG4gICAgY29uc3QgY29udGFpbnNDaGVja2VkID0gX2RhdGEuc29tZSgocm93KSA9PiByb3cuY2hlY2tlZClcblxuICAgIHJldHVybiAoXG4gICAgICA8U2VsZWN0aW9uQ2VsbFxuICAgICAgICB2YWx1ZT17YWxsU2VsZWN0ZWR9XG4gICAgICAgIGludGVybWVkaWF0ZT17Y29udGFpbnNDaGVja2VkICYmICFhbGxTZWxlY3RlZH1cbiAgICAgICAgb25DaGFuZ2U9e29uQ2hhbmdlfVxuICAgICAgLz5cbiAgICApXG4gIH0sXG59KVxuXG5jb2x1bW5zLnNwbGljZShjb2x1bW5zLmxlbmd0aCAtMiwgMCx7XG4gIGtleTogJ3NlbGVjdGlvbicsXG4gIHdpZHRoOiA1MCxcbiAgXG4gIGNlbGxSZW5kZXJlcjogKHsgcm93RGF0YSB9KSA9PiB7XG4gICAgY29uc3Qgb25DaGFuZ2UgPSAodmFsdWU6IENoZWNrYm94VmFsdWVUeXBlKSA9PiAocm93RGF0YS5jaGVja2VkID0gdmFsdWUpXG4gICAgcmV0dXJuIDxTZWxlY3Rpb25DZWxsIHZhbHVlPXtyb3dEYXRhLmNoZWNrZWR9IG9uQ2hhbmdlPXtvbkNoYW5nZX0gLz5cbiAgfSxcblxuICBoZWFkZXJDZWxsUmVuZGVyZXI6ICgpID0+IHtcbiAgICBjb25zdCBfZGF0YSA9IHVucmVmKGRhdGEpXG4gICAgY29uc3Qgb25DaGFuZ2UgPSAodmFsdW)

**Review by btea — APPROVED** [review]
Thanks!

File: packages/theme-chalk/src/table-v2.scss (hunk 1 above)
[the remark this answers is not in the document]
Comment (warmthsea) [reply packages/theme-chalk/src/table-v2.scss]: @btea 

Maybe we should use:

```scss
@include e('header') {
    .#{$namespace}-checkbox {
      z-index: 0;
    }
}
```

about: https://github.com/element-plus/element-plus/issues/18550

**Pull request merged**